packages/playground/wordpress/src/index.ts

```diff
@@ -1,3 +1,107 @@
 export * from './rewrite-rules';
 
 export const RecommendedPHPVersion = '8.0';
+
+export const envPHP_to_loadMuPlugins = `<?php
```

Comment:
Why store this PHP code in JS? Would it be possible to load it from PHP files?

Reply:
I couldn't figure out a way to do it that would work with all the build systems (Bun, ESBuild, Rollup).

Reply:
Sometimes, it's nice to be able to read see everything inline anyway.